src/docfx/build/metadata/UserMetadata.cs

```diff
@@ -52,6 +52,10 @@ internal class UserMetadata
 
         public string? OriginalContentGitUrlTemplate { get; private set; }
 
+        public string? ZonePivotGroupFilename { get; private set; }
```

Comment:
Add a comment that this is the file name of the published URL with .json extension, not the source file name.

See an example [here](https://github.com/dotnet/AspNetCore.Docs/blob/93f237ef4ddf9f68ab29bd7a1ba6f9e2c6904a36/aspnetcore/docfx.json#L60)

[This](https://dev.azure.com/ceapex/Engineering/_git/docs-ui?path=%2Fpackages%2Fscripts%2Fsrc%2Fcomponents%2Fzone-pivot.ts&version=GBmaster&line=310&lineEnd=310&lineStartColumn=17&lineEndColumn=34&lineStyle=plain) is how docs-ui loads the zone pivot files.

Adjust the implementation accordingly to load the zone pivot definition file based on publish URL. You can get the publish URL from `PublishUrlMap.GetFilesByUrl` method.